files/image_config/monit/container_checker

```diff
@@ -70,25 +51,60 @@ def get_expected_running_containers():
                     num_asics = multi_asic.get_num_asics()
                     for asic_id in range(num_asics):
                         expected_running_containers.add(container_name + str(asic_id))
+            elif feature_table[container_name]["state"] == 'always_enabled':
+                always_running_containers.add(container_name)
             else:
                 expected_running_containers.add(container_name)
 
-    return expected_running_containers
+    return expected_running_containers, always_running_containers
 
 
-def get_current_running_containers():
+def get_current_running_from_DB(always_running_containers):
     """
-    @summary: This function will get the current running container list by analyzing the
-              output of command `docker ps`.
-    @return:  A set which contains the current running contianers.
+    @summary: This function will get the current running container list
+              from FEATURE table @ STATE_DB, if this table is available.
+    @return:  a tuple
+              First: Return value indicating if info can be obtained from
+              DB or not.
+              Second: A set which contains the current running containers,
+              if this info is available in DB
     """
     running_containers = set()
 
-    command = "docker ps"
-    command_stdout = get_command_result(command)
-    for line in command_stdout[1:]:
-        running_containers.add(line.split()[-1].strip())
+    state_db = swsscommon.DBConnector("STATE_DB", 0)
+    tbl = swsscommon.Table(state_db, "FEATURE")
+    if not tbl.getKeys():
+        return False, None
+
+    for name in tbl.getKeys():
```

Comment:
One question: whether the `container_name` in `always_running_containers` is included in `tbl.getKeys()`?

Reply:
rest, except "database".

Reply:
That's why we still need the code from line 84 to 90, right?

Reply:
yes. 
I could do some optimizations, but realized not worth it.
In fact being generic - helps handle any newer scenarios too, transparently.